topology/probes/k8s/hostnodelinker.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/skydive-project/skydive/probe"
+	"github.com/skydive-project/skydive/topology/graph"
+)
+
+func newHostNodeLinker(g *graph.Graph, subprobes map[string]*ResourceCache) probe.Probe {
+	nodeProbe := subprobes["node"]
+	if nodeProbe == nil {
```

Comment:
but on line 33 you initialized `nodeProbe` thus `nodePorbe != nil`

Reply:
If the "node" probe is not enabled, nodeProbe would be "nil"

Reply:
ok